service/telemetry/config.go

```diff
@@ -93,19 +93,6 @@ type LogsSamplingConfig struct {
 	Thereafter int `mapstructure:"thereafter"`
 }
 
-// MetricReader exposes configuration of metric readers to end users.
-// TODO: replace this temporary struct w/ auto-generated struct from jsonschema
-// https://github.com/open-telemetry/opentelemetry-configuration/tree/main/schema
-//
-// Experimental: *NOTE* this structure is subject to change or removal in the future.
-type MetricReader struct {
-	// Args corresponds to the JSON schema field "args".
-	Args any `mapstructure:"args"`
-
-	// Type corresponds to the JSON schema field "type".
-	Type string `mapstructure:"type"`
```

Comment:
Given that these fields were released previously and removed now, should we still mark it as a breaking change even if it's marked as experimental?

Reply:
Added a changelog.